package.json

```diff
@@ -23,10 +23,15 @@
     "@nestjs/common": "^10.0.0",
     "@nestjs/core": "^10.0.0",
     "@nestjs/platform-express": "^10.0.0",
+    "@nestjs/swagger": "^7.4.0",
     "@nestjs/typeorm": "^10.0.2",
+    "class-transformer": "^0.5.1",
+    "class-validator": "^0.14.1",
+    "dotenv": "^16.4.5",
```

Comment:
**issue:** i see you added the dotenv dependency but are not using in your app.module to get the relevant environment variables to setup your connection to your database (e.g.: host, port, username, password, etc).

Link: https://docs.nestjs.com/techniques/configuration
Link: https://docs.nestjs.com/techniques/database#async-configuration

Reply:
I wasn't sure why getting the PORT for the app as an environment variable wasn't working. Thank you for pointing me to the relevant documentation.
I fixed it like this:
`ConfigModule.forRoot({
      isGlobal: true,
      envFilePath: `.env.${(process.env.NODE_ENV as string) || 'dev'}`,
      cache:true
    }),
    TypeOrmModule.forRootAsync({
      imports: [ConfigModule],
      useFactory: (configService: ConfigService) => ({
        type: 'postgres',
        host: configService.get('DB_HOST'),
        port: +configService.get('DB_PORT'),
        username: configService.get('DB_USERNAME'),
        password: configService.get('DB_PASSWORD'),
        database: configService.get('DB_NAME'),
        entities: [],
        synchronize: true,
      }),
      inject: [ConfigService],
    }),`